controller/deployments.go

```diff
@@ -262,8 +262,11 @@ func (g *defaultClientGetter) GetKubeClient(ctx context.Context) (kubernetes.Kub
 	/* Timeout used per HTTP request to Kubernetes/OpenShift API servers.
 	 * Communication with Hawkular currently uses a hard-coded 30 second
 	 * timeout per request, and does not use this parameter. */
-	// TODO replace with configurable timeout
-	const timeout time.Duration = 30 * time.Second
+	registry, err := configuration.Get()
+	if err != nil {
+		return nil, err
+	}
+	timeout := time.Duration(registry.GetDeploymentsHttpTimeout()) * time.Second
```

Comment:
For style here, I suggest time.Duration(registry.GetDeploymentsHttpTimeout() * time.Second) - i.e. create the Duration once using the correct value.